sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/SignatureAlgorithm.cs

```diff
@@ -24,66 +35,56 @@ public SignatureAlgorithm(string value)
         {
             _value = value ?? throw new ArgumentNullException(nameof(value));
         }
-        internal const string RS256 = "RS256";
-        internal const string RS384 = "RS384";
-        internal const string RS512 = "RS512";
-        internal const string PS256 = "PS256";
-        internal const string PS384 = "PS384";
-        internal const string PS512 = "PS512";
-        internal const string ES256 = "ES256";
-        internal const string ES384 = "ES384";
-        internal const string ES512 = "ES512";
-        internal const string ES256K = "ES256K";
 
         /// <summary>
-        /// RSA SHA-256 signature algorithim.
+        /// RSA SHA-256 signature algorithm.
         /// </summary>
-        public static readonly SignatureAlgorithm Rs256 = new SignatureAlgorithm(RS256);
+        public static readonly SignatureAlgorithm RS256 = new SignatureAlgorithm(RS256Value);
```

Comment:
I find it kind of weird that this is all caps when the other algorithms are camel cased.  I know the difference here is that these are two letters vs. three, but it makes it look inconsistent.  I not insisting that we change it I just wanted to start a conversation about it.

Reply:
We did agree on the example of "A128KW", for example, because it follows both the 1 and 2 character rule. Granted, we removed those for now. @KrzysztofCwalina, thoughts?

Reply:
Given the conversation we had internally, since the "R" and "S" in this case are from standalone acronyms ("RSA Signature"), our current guidelines (with Krzysztof's clarification) apply here that we use "RS".